src/dali/plugin/input/csv/binance_com.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2022 macanudo527
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Autoinvest CSV Format: timestamp UTC, base asset symbol, quote asset amount + symbol, trading fee (in quote asset),
+#    base asset amount + symbol, source of funds
+# Note: file comes as .xlsx, and then needs to be saved as CSV.
+
+# Betheth CSV format: timestamp UTC, quote asset symbol (ETH), base asset symbol (BETH), amount, status
+
+import logging
+from csv import reader
+from typing import List, Optional
+
+from rp2.logger import create_logger
+from rp2.rp2_decimal import RP2Decimal
+
+from dali.abstract_input_plugin import AbstractInputPlugin
+from dali.abstract_transaction import AbstractTransaction
+from dali.configuration import Keyword
+from dali.in_transaction import InTransaction
+from dali.out_transaction import OutTransaction
+
+
+class InputPlugin(AbstractInputPlugin):
+
+    __BINANCE_COM: str = "Binance.com"
+    __BINANCE_COM_CSV: str = "Binance.com CSV"
```

Comment:
Also add "supplemental" both to variable name and to string

Reply:
Maybe remove substitute white space with underscore, because this will be part of a file system path (see log file, line 61)